pom.xml

```diff
@@ -17,6 +18,21 @@
     <tag>HEAD</tag>
   </scm>
 
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/LiveRamp/daemon_lib/issues</url>
+  </issueManagement>
+
+  <url>https://liveramp.com</url>
+
+  <licenses>
+    <license>
+      <name>MIT</name>
```

Comment:
I don't care one way or the other, but we should stay consistent on the licensing -- LICENSE.txt here is Apache 2.0

Reply:
Actually, doing some brief research, I think the patent troll protections in Apache 2.0 seem nice.  Might be worth bringing up with the arch board?  I am leaning Apache though.

Reply:
I'll stick with what's checked in for now. Thanks for the heads up